app/audio/microphone.js

```diff
@@ -13,7 +13,7 @@ export default function() {
     const analyzer = context.createAnalyser();
     analyzer.fftSize = 2048;
     let dataArray = new Uint8Array(analyzer.frequencyBinCount);
-    analyzer.getByteTimeDomainData(dataArray);
+    let frequency = new Uint8Array(analyzer.frequencyBinCount);
```

Comment:
Yes, but it seems that calling `analyzer.getByteFrequencyData` actually modifies the argument that is passed in, so I called `new Uint8Array` to make a new copy, to leave the dataArray unchanged. I *might* be wrong.